src/stitching/defaultMergedResolver.ts

```diff
@@ -0,0 +1,35 @@
+import { GraphQLFieldResolver, responsePathAsArray } from 'graphql';
+import { locatedError } from 'graphql/error';
+import { getErrorsFromParent, annotateWithChildrenErrors } from './errors';
+
+// Resolver that knows how to:
+// a) handle aliases for proxied schemas
+// b) handle errors from proxied schemas
+const defaultMergedResolver: GraphQLFieldResolver<any, any> = (
+  parent,
+  args,
+  context,
+  info,
+) => {
+  const fieldName = info.fieldNodes[0].alias
+    ? info.fieldNodes[0].alias.value
+    : info.fieldName;
+  const { ownError, childrenErrors } = getErrorsFromParent(parent, fieldName);
+  if (ownError) {
+    throw locatedError(
+      ownError.message,
```

Comment:
This is not for now I think, but we may want to make sure we can also support additional error fields besides message.

Reply:
So the problem is the implementation of locatedError, it copies all the fields like "location" and "path" that we actually want to recreate.